pulumi/manifest.json

```diff
@@ -0,0 +1,37 @@
+{
+  "display_name": "Pulumi",
+  "maintainer": "team@pulumi.com",
+  "manifest_version": "1.0.0",
+  "name": "pulumi",
+  "metric_prefix": "pulumi.",
+  "metric_to_check": "",
+  "creates_events": false,
+  "short_description": "Infrastructure as code for any cloud using your favorite programming languages",
+  "guid": "58481132-c79e-4659-8064-7cdaabbbc981",
+  "support": "core",
```

Comment:
```suggestion
  "support": "contrib",
```

Reply:
@ChristineTChen I think I've addressed this -- please let me know if there are other changes needed.